ipn/prefs.go

```diff
@@ -24,6 +25,7 @@ type Prefs struct {
 	WantRunning      bool
 	NotepadURLs      bool
 	UsePacketFilter  bool
+	AdvertiseRoutes  []*net.IPNet
```

Comment:
Start adding some godoc while you're here? It wasn't great before, but adding a field makes it worse.